app/modules/toolkits/rdkit_wrapper.py

```diff
@@ -145,8 +145,8 @@ def get_tanimoto_similarity_rdkit(
         mol1 (Chem.Mol): The RDKit Mol object representing the first molecule.
         mol2 (Chem.Mol): The RDKit Mol object representing the second molecule.
         fingerprinter (str, optional): The type of fingerprint to use. Defaults to "ECFP".
```

Comment:
fingerprinter (str, optional): The type of fingerprint to use. *Options are "ECFP", "RDKit", "AtomPairs", "MACCS".* Defaults to "ECFP".